docs/tutorials/simple_trading.py.md

```diff
@@ -84,11 +84,11 @@ class SimpleMarketListener(MarketStreamListener):
     futures = [instrument for instrument in instrument_data['data'].values() if instrument['type'] == 'futures'][0]
     selected_futures_id = futures['instrument_id']
 
-  def on_order_book(self, order_book):
+  def on_order_book(self, order_book):  
     if order_book['instrument_id'] != selected_futures_id:
       return 
     bids = order_book['bids']
-    if bids and (not bids[0] or bids[0][0] > sell_threshold):
+    if bids and (not bids[0] or float(bids[0][0]) > sell_threshold):
```

Comment:
Ok, I will be adding: defensiveness, reconnecting, basic error handling to the tutorial.